modules/sdk/gradle-plugins/src/main/java/com/liferay/gradle/plugins/tasks/WatchTask.java

```diff
@@ -525,7 +525,11 @@ private boolean _isManifestChanged(List<File> modifiedFiles)
 		while (iterator.hasNext()) {
 			Map.Entry<Object, Object> entry = iterator.next();
 
-			if (_ignoredManifestKeys.contains(entry.getKey())) {
+			Object key = entry.getKey();
+
+			String keyName = key.toString();
```

Comment:
Easier to do `String key = String.valueOf(entry.getKey())`, what do you think? 😊